src/common/types/store.ts

```
@@ -54,11 +53,22 @@ const variantFriendlyName = z.string().min(1).max(30).optional().meta({
   examples: ["Size", "Partner Organization"]
 });
 
+const requestingImageUpload = z.optional(z.object({
```

ESLint

[error] Replace `.optional(z` with `⏎··.optional(⏎····z⏎······`

(prettier/prettier)